lib/model/question_dislplay_type_model.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:survey_flutter_ic/api/response/question_display_type_response.dart';
+
+enum QuestionDisplayType {
+  singleChoice,
+  multipleChoice,
+  text,
+  number,
+  email,
+  dateTime,
+  image,
+  video,
+  audio,
```

Comment:
@hoangnguyen92dn Can I know where you obtain these types? 🙏 I haven't seen these question types before 🤔 `image, dateTime, video, audio`

Reply:
@manh-t Ops 🙈 Codepilot applied 🤖